examples/browserified/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel='stylesheet' id='source-sans-pro-css'
           href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600&amp;subset=latin'/>
     <link rel="stylesheet" href="./style.css">
-    <link rel="stylesheet" href="../../dist/yoast-seo.min.css">
+    <link rel="stylesheet" href="../shared/yoast-seo.min.css">
```

Comment:
Why this change?

Reply:
It was in the dist folder, which is a more logical to place the built files. Since this css files is only used in the examples I moved it to a shared folder in the examples folder.